pages/login/login.css

```diff
@@ -0,0 +1,73 @@
+.navbar {
+  justify-content: space-between;
+}
+.navbar .btn {
+  margin: 0;
+  background-color: var(--primary-color);
+}
+.navbar .right-nav {
+  display: inline-flex;
+  margin: 0;
+}
+.navbar .navbar-btn {
+  display: none;
+}
+@media (max-width: 768px) {
```

Comment:
it would be better if u write media queries at the end of the CSS file. because nothing can override it.

Reply:
Done